libraries/common/include/common/constants.hpp

```diff
@@ -19,6 +19,11 @@ constexpr uint16_t kOnePercent = 100;
 constexpr uint16_t kMaxLevelsPerPeriod = 100;
 constexpr uint32_t kDagExpiryLevelLimit = 1000;
 
+const uint32_t kMaxVotesInPacket{1000};
+const uint32_t kMaxTransactionsInPacket{1000};
+
+const uint32_t kPeriodicEventsThreadCount{2};
```

Comment:
so what is the purpose/idea behind this common constants file ? Because all of these new constants are not shared between multiple files/modules. Those are specific constants used on one specific file and could be part of the headers

Reply:
I am fine either way. I am not sure when we introduced this file but I do like to have similar constants grouped. For instance kMaxVotesInPacket and  kMaxTransactionsInPacket I would put in the same file but maybe it should be in somewhere under network. I think that it would be good to have constants defining number of threads used in different places also to be grouped so that we can easily see how many threads we actually startup in our node but I can see arguments against it as well.